liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/DevMojo.java

```diff
@@ -732,6 +732,9 @@ private void addDomPropertyIfNotFound(Xpp3Dom sysProps, String key, String value
             if (testServerName != null) {
                 elements.add(element(name("serverName"), testServerName));
                 elements.add(element(name("configDirectory"), configDirectory.getCanonicalPath()));
+                if (installDirectory != null && installDirectory.exists()) {
+                    elements.add(element(name("installDirectory"), installDirectory.getCanonicalPath()));
+                }
```

Comment:
Are there other common parameters that need to be checked for to? Or are we just going to add them as people report having an issue with them?

Reply:
Opened https://github.com/OpenLiberty/ci.maven/issues/558 to address the other parameters.